sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/actions/CreateNewActionSteps.java

```diff
@@ -56,7 +55,7 @@ public CreateNewActionSteps(
           fillDescription("Dummy description for automated test");
           selectActionStatus(action.getActionStatus());
           webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
-          webDriverHelpers.waitUntilIdentifiedElementIsPresent(UUID_INPUT);
+          // webDriverHelpers.waitUntilIdentifiedElementIsPresent(UUID_INPUT);
```

Comment:
please remove this commented line in the next commit. I'll approve the PR but don't forget about it

Reply:
addressed.